src/DatadogLoggingService.js

```diff
@@ -35,8 +35,8 @@ class DatadogLoggingService extends NewRelicLoggingService {
       service: process.env.DATADOG_SERVICE,
       env: process.env.DATADOG_ENV,
       version: process.env.DATADOG_VERSION,
-      sessionSampleRate: 50,
-      sessionReplaySampleRate: 100,
+      sessionSampleRate: parseInt(process.env.SESSION_SAMPLE_RATE || 0, 10),
+      sessionReplaySampleRate: parseInt(process.env.SESSION_REPLAY_SAMPLE_RATE || 0, 10),
```

Comment:
0 default seems right for this for now. Should we add a comment that we are defaulting to 0 until we work out privacy concerns?